src/main/java/com/icegreen/greenmail/imap/commands/CreateCommand.java

```diff
@@ -27,7 +31,7 @@ protected void doProcess(ImapRequestLineReader request,
                              ImapSession session)
             throws ProtocolException, FolderException, AuthorizationException {
         String mailboxName = parser.mailbox(request);
-        parser.endLine(request);
+//        parser.endLine(request);

```

Comment:
No, i can't, because of the extra Cyrus parameter (partition).
The goal of this commit is to make the IMAP CREATE command compatible with Cyrus.